README.md

```diff
@@ -44,3 +49,52 @@ A [FastAPI](https://fastapi.tiangolo.com/) server is running in the container. I
 ### Frontend
 
 A minimal frontend lets the user input a question and renders the response from the system.
+
+## Sentence Transformers Statistics
+
+```
```

Comment:
I am happy that we are keeping this information somewhere. 

It took me a while to curate the list of viable German-aware models that were both sentence-transformers compatible + haystack2beta compatible.

At the same time, I am not sure that README.md is the best place. 

What do we think about "research_log.md"?

Reply:
Like the suggestion, will move it there.

Reply:
done in [7a2c955](https://github.com/rti/gbnc/pull/23/commits/7a2c9553f668d60aa964f84fae0f6ecc050a271f)